scripts/release/asset_publish.py

```diff
@@ -138,6 +138,64 @@ def prepare_model(model_config: assets.ModelConfig, spec_file_path: Path, model_
     return can_publish_model
 
 
+def validate_update_command_component(
+    mlclient: MLClient,
+    component: Component,
+    spec_path: Path,
+    final_version: str
+) -> bool:
+    """Validate and update command component spec.
+
+    :param mlclient: MLClient object
+    :type mlclient: MLClient
+    :param component: A command component
+    :type component: Component
+    :param spec_path: Path of loaded component
+    :type spec_path: Path
+    :param final_version: Final version string used to register component
+    :type final_version: str
+    :return: True for successful validation and update
+    :rtype: bool
+    """
+    env = component.environment
+    match = None
+    for pattern in [REGISTRY_ENV_PATTERN, WORKSPACE_ENV_PATTERN]:
+        if (match := pattern.match(env)) is not None:
+            break
+
+    if not match:
+        logger.print(f"Env ID doesn't match workspace or registry pattern in {asset.spec_with_path}")
+        return False
+
+    # both ws and registry env follows the same grouping
+    env_name, env_version, env_label = match.group(1), match.group(2), match.group(3)
+    logger.print(f"Env name: {env_name}, version: {env_version}, label: {env_label}")
+
+    if env_label:
+        # TODO: Add fetching env from label
+        # https://github.com/Azure/azureml-assets/issues/415
+        logger.print("Unexpected !!! Registering a component with env label is not yet supported.")
```

Comment:
I would remove `yet` to not set any expectations

Reply:
:D, removed.